lib/rules/prefer-true-attribute-shorthand.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @author Pig Fang
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const utils = require('../utils')
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    type: 'problem',
```

Comment:
It's only a stylistic/conventional choice, not a problem.

```suggestion
    type: 'suggestion',
```